webapp/src/components/ProgramActivityDialog.tsx

```diff
@@ -76,9 +76,12 @@ const ProgramActivityDialog = ({
           {timeRange()}
         </DialogContentText>
 
+        <DialogContentText><b>Program:</b>{contents.theProgram}</DialogContentText>
```

Comment:
I don't know how I didn't catch it before and that it got through PR, but in HTML we use `<strong></strong>` for emboldened emphasis; the use of `<b></b>` is either deprecated or discouraged.

If you end up taking Wendy's suggestion and adding a space, please make sure it's _after_ the `</strong>` and not before.